admin/src/app/search/search.component.html

```diff
@@ -25,13 +25,27 @@ <h1 class="text-muted">Search FOMs</h1>
   <div class="bottom-container scroll">
     <div class="container" style="background: #f7f8fa; padding-bottom: 0;">
       <div class="row">
-        <div class="form-group col-md-1">
+        <div class="form-group col-md-2">
+          <label for="fomNumberInput" style="white-space: nowrap;">
+                FOM Number
+          </label>
+
+          <input id="fNumberInput" name="fNumber" type="number"
+                min="0" step="1"
+                oninput="this.value = this.value.slice(0, 9)"
+                class="form-control"
+                style="font-size: 14px;"
+                [(ngModel)]="fNumber"
+                [disabled]="searching" autofocus/>
+        </div>
+
+        <div class="form-group col-md-2">
           <label for="fspIdInput" style="white-space: nowrap;">
                 FSP ID
           </label>
           <input id="fspIdInput" name="fFspId" type="number"
                 min="0" step="1"
-                oninput="javascript: if (this.value.length > this.maxLength) this.value = this.value.slice(0, 9);"
+                oninput="this.value = this.value.slice(0, 9)"
```

Comment:
It works previously, I only remove unnecessary 'javascript' part and condition.
New FOM Number field also follows this one.
I could not find a better way to be close to Angular way for event handling and assign new value to view value in this particular case. This is probably best I can see so far.
The only case it seems not working is entering multiple '0'(s), but I feel we can ignore this problem for now...

Reply:
Hi Ian, I'm not sure if I understand correctly, but when I put a number longer than 10 chars, I got an error
<img width="1261" alt="Screen Shot 2024-07-30 at 10 25 29 AM" src="https://github.com/user-attachments/assets/85077ad4-5594-4ed4-aab0-68b07705b4d4">

Reply:
Interesting, maybe I only verify locally, thanks, let me check on why it is allowing fore more digits.

Reply:
I see, maybe we could create another ticket to check this one?

Reply:
Spent a little time, can't seem to make <input type="number"> to restrict digits with event listener to work.
I seem to recall Basil mentioned before to use <input type="text"> is better (however, I do see some minor drawback using 'text' for integer field) but can work.
Will ignore this for now and create new ticket to deal with it (similar like public side or using some sort of Angular directive for better input experience).